<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43);
font: 12pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>427</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Century Gothic&quot;;
color: rgb(200, 200, 200);
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Emotion Predominantly: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Century Gothic&quot;;
color: rgb(200, 200, 200);
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Taste Predominantly: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidget_taste">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidget_emotion">
      <property name="styleSheet">
       <string notr="true">font-weight: regular;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
